src/core/gltf2importer/meshparser_utils.cpp

```diff
@@ -376,8 +376,31 @@ bool vertexSizesForAttributesAreValid(const QVector<Qt3DRender::QAttribute *> &a
 namespace Kuesa {
 namespace GLTF2Import {
 namespace MeshParserUtils {
-Qt3DRender::QAttribute *createTangentAttribute(Qt3DRender::QGeometry *geometry, Qt3DRender::QGeometryRenderer::PrimitiveType primitiveType)
+
+Qt3DRender::QAttribute *createNewTangentAttributeIfNeeded(Qt3DRender::QGeometryRenderer *mesh)
 {
+    const auto primitiveType = mesh->primitiveType();
+
+    // only generate tangents for triangles
+    if (primitiveType != Qt3DRender::QGeometryRenderer::Triangles
+            && primitiveType != Qt3DRender::QGeometryRenderer::TriangleStrip
```

Comment:
indent seems wrong, was clang-format run?

Reply:
fixed, indented it in a way that clang-format is happy with (and how it was before), but I don't actually see the point here. Why would I want to indent it in a way that makes it impossible to distinguish if() condition and the resulting statement, plus not even be able to figure out by looking at the first characters whether this is a || or && ?